assets/css/screen.css

```diff
@@ -453,23 +454,141 @@ The knock-on effect of this is ugly browser-scroll bars at the bottom, so 80px o
 }
 
 .post-card-meta {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
     padding: 0 25px 25px;
 }
 
+.profile-image-wrapper,
+.avatar-wrapper {
+    position: absolute;
+    display: inline-block;
+    width: 28px;
+}
+
+.avatar-wrapper {
+    display: inline-block;
+    text-align: center;
+    background-color: color(var(--lightgrey) l(+10%));
+    border-radius: 100%;
+    border: 2px solid #fff;
+    box-sizing: content-box;
+    margin-right: 3px;
+}
+
+.avatar-wrapper svg {
+    width: 28px;
+    height: 28px;
+}
+
 .author-profile-image {
     margin-right: 5px;
-    width: 25px;
-    height: 25px;
+    width: 28px;
+    height: 28px;
     border-radius: 100%;
+    border: 2px solid #fff;
+    box-sizing: content-box;
 
     object-fit: cover;
 }
 
-.post-card-author {
-    font-size: 1.3rem;
+.post-card-meta > .avatar-wrapper {
+    position: relative;
+}
+
+.author-list {
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    margin-bottom: 2em;
+    max-width: calc(100% - 120px);
+}
+
+.author-list .author-list-item {
```

Comment:
This (and all below) is probably too specific. Is there any instance of .author-list-item outside of .author-list? If not, then it doesn't need to be scoped to .author-list